user-access-commons/src/main/java/no/unit/nva/database/DatabaseServiceImpl.java

```diff
@@ -169,17 +192,33 @@ private static boolean isInvalid(Validable roleDto) {
         return isNull(roleDto) || !roleDto.isValid();
     }
 
+    private static RuntimeException logErrorWithDynamoClientAndThrowException(Failure<AmazonDynamoDB> failure) {
```

Comment:
Implementation here does return a RuntimeException, but throws it.  Make void?

Reply:
nice catch